docs/user/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -25,10 +25,10 @@ Add the following setting in the {kib} configuration file (`kibana.yml`):
 
 [source,yaml]
 ----------------------------------
-xpack.monitoring.kibana.collection.enabled: false
+monitoring.kibana.collection.enabled: false
 ----------------------------------
 
-Leave the `xpack.monitoring.enabled` set to its default value (`true`). 
+Leave the `monitoring.enabled` set to its default value (`true`). 
 // end::disable-kibana-collection[]
 For more information, see 
 <<monitoring-settings-kb,Monitoring settings in {kib}>>.
```

Comment:
I see more instances in monitoring-metricbeat-asciidoc

<img width="628" alt="Screen Shot 2020-04-20 at 9 37 09 AM" src="https://user-images.githubusercontent.com/33642766/79776428-94492080-82ea-11ea-9292-fb03274c3e62.png">

Reply:
Same as above